.github/workflows/test-lightwalletd.yml

```diff
@@ -0,0 +1,370 @@
+  lightwalletd-full-sync:
+    name: Test lightwalletd full sync
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Run tests with lightwalletd full sync
+        run: |
+          docker pull ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+          docker run -e ZEBRA_SKIP_IPV6_TESTS -e ZEBRA_TEST_LIGHTWALLETD -e ZEBRA_CACHED_STATE_PATH -e LIGHTWALLETD_DATA_DIR --name lightwalletd-tests -t ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} cargo test --locked --release --features enable-sentry -- --ignored --nocapture  lightwalletd_full_sync
+        env:
+          ZEBRA_CACHED_STATE_PATH: '$HOME/.cache/zebra'
+          LIGHTWALLETD_DATA_DIR: '$HOME/.cache/lightwalletd'
+
+  # TODO: add description to this test and verify name
+  lightwalletd-update-sync:
+    name: Test lightwalletd update
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Run tests with lightwalletd update sync
+        run: |
+          docker pull ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+          docker run -e ZEBRA_SKIP_IPV6_TESTS -e ZEBRA_TEST_LIGHTWALLETD -e ZEBRA_CACHED_STATE_PATH -e LIGHTWALLETD_DATA_DIR --name lightwalletd-tests -t ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} cargo test --locked --release --features enable-sentry -- --ignored --nocapture  lightwalletd_update_sync
+        env:
+          ZEBRA_CACHED_STATE_PATH: '$HOME/.cache/zebra'
+          LIGHTWALLETD_DATA_DIR: '$HOME/.cache/lightwalletd'
+
+  # TODO: add description to this test and verify name
+  lightwalletd-test-suite:
+    name: Test lightwalletd suite
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Run tests with lightwalletd suite
+        run: |
+          docker pull ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+          docker run -e ZEBRA_SKIP_IPV6_TESTS -e ZEBRA_TEST_LIGHTWALLETD -e ZEBRA_CACHED_STATE_PATH -e LIGHTWALLETD_DATA_DIR --name lightwalletd-tests -t ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} cargo test --locked --release --features enable-sentry -- --ignored --nocapture  lightwalletd_test_suite
+        env:
+          ZEBRA_CACHED_STATE_PATH: '$HOME/.cache/zebra'
+
+  # TODO: add description to this test and verify name
```

Comment:
@conradoplg is probably the best person to review the fully synced RPC test

Reply:
I made the env vars in this test consistent with the other tests in PR #4177

Reply:
Fixed by [fix(ci): use correct lwd variables for tests](https://github.com/ZcashFoundation/zebra/pull/4223/commits/bfd76b7f29b15204dde9eaf550e8cc84d4c25dcf)